source/common/tls/tls_certificate_selector_impl.cc

```diff
@@ -0,0 +1,273 @@
+#include "source/common/tls/tls_certificate_selector_impl.h"
```

Comment:
I think this file should be renamed `default_tls_certificate_selector.cc` (and header similarly)

Reply:
yes, just simply code moving.